output.js

```diff
@@ -4,6 +4,7 @@ const assert = require('assert');
 const readline = require('readline');
 
 const utils = require('./utils');
+const {resultCountString} = require('./results');
```

Comment:
`./results` doesn't exist in the codebase. Maybe you forgot to include that file in your commit? 🙈

Reply:
woops, fixed. The problem was that `results*` was gitignored :scream: